Docs/wallet.md

```diff
@@ -20,140 +29,20 @@ A class for performing various operations on a wallet.
 <a name="new_module_nahmii-sdk--Wallet_new"></a>
 
 #### new Wallet(signer, provider)
-
-Create a Wallet from a private key or custom address and signing parameters.
-
-Use a private key to initialize a software wallet, and custom parameters to initialize a hardware wallet.
-
-The custom parameters are
-
-- {function} signMessage - Takes a string as input and returns a flat format Ethereum signature
-- {function} signTransaction - Takes a transaction as input and returns the same transaction signed as a hex string. The input may contain unresolved values, so it's important to wait for them to resolve with `ethers.utils.resolveProperties(tx);` before performing any logic.
-- {string} address - The address to use. Must be able to derive from the private key used in the signing functions
-
+Create a Wallet from either a private key or custom signing functions
 
 
 | Param | Type | Description |
 | --- | --- | --- |
-| signer | <code>string</code> or <code>object</code> | A private key, or object containing custom parameters |
+| signer | <code>string</code> \| <code>object</code> | A private key, or information required for the wallet to have signing capabilities |
 | provider | <code>NahmiiProvider</code> | A NahmiiProvider instance |
 
-**Wallet from private key example**
-```js
-const privateKey = '0x9616c2ab6330c7fda535042c120b55d992fa8c2c2a3d82603ea043aeb09ff411';
-const provider = new NahmiiProvider(...);
-const softwareWallet = new Wallet(privateKey, provider);
-```
-
-**Wallet from Ledger Nano S example**
```

Comment:
The hardware wallet examples are removed. Should we create maybe a new .md file with these examples?

Reply:
Docs/*.md are generated files from JSDOC comments. Generation should be added to the npm "build" script.

Reply:
For large code examples would it makes sense to have a separate folder called "examples"?